commit 42c9a288d10c6d5f6aa850478bd7cf7dcbae8e8c (refs/changes/07/5707/1)
Author: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
Date:   2019-03-04T15:22:58-07:00 (7 months ago)
    
    MANTA-4148 Make boray work on SmartOS with unreleased rand crate version

diff --git a/Cargo.toml b/Cargo.toml
index b5df275..8d1fbfb 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -20,3 +20,6 @@ slog = { version = "2.4.1", features = [ "max_level_trace" ] }
 slog-bunyan = { git = "https://github.com/kellymclaughlin/bunyan", branch = "build-on-smartos" }
 tokio = "0.1.14"
 uuid = { version = "0.5", features = ["serde", "v4"] }
+
+[patch.crates-io]
+rand = { git = 'https://github.com/rust-random/rand' }
